src/global_legacy_module/4C_global_legacy_module_validmaterials.cpp

```diff
@@ -2923,20 +2925,22 @@ std::shared_ptr<std::vector<std::shared_ptr<Mat::MaterialDefinition>>> Global::v
   /*----------------------------------------------------------------------*/
   {
     auto m = std::make_shared<Mat::MaterialDefinition>("MAT_ViscoplasticLawReformulatedJohnsonCook",
-        "Reformulation of the Johnson-Cook viscoplastic law (comprising flow rule and hardening "
+        "Reformulation of the Johnson-Cook viscoplastic law (comprising flow rule \f$\\dot{P} = "
+        "\\dot{P}_0 \\exp \\left( \\frac{ \\Sigma_{eq}}{C \\Sigma_y} - \\frac{1}{C} \\right) - "
+        "\\dot{P}_0\f$ and hardening "
         "law), as shown in Mareau et al. (Mechanics of Materials 143, 2020)",
         Core::Materials::mvl_reformulated_Johnson_Cook);
 
     m->add_component(entry<double>(
-        "STRAIN_RATE_PREFAC", {.description = "plastic strain rate prefactor $\\dot{P}_0$"}));
-    m->add_component(entry<double>(
-        "STRAIN_RATE_EXP_FAC", {.description = "exponential factor of plastic strain rate $C$"}));
-    m->add_component(entry<double>(
-        "INIT_YIELD_STRENGTH", {.description = "initial yield strength of the material $A_0$"}));
+        "STRAIN_RATE_PREFAC", {.description = "reference plastic strain rate \f$\\dot{P}_0\f$ "}));
```

Comment:
```suggestion
        "STRAIN_RATE_PREFAC", {.description = "reference plastic strain rate $\\dot{P}_0$ "}));
```

`\f` is the form feed character that is not printable. Why do you insert it here?  I suspect because doxyggen uses it, but this is not a doxygen interpreted comment. Such illegal character will get sanitized in the future by `InputSpec` automatically and will have no effect.

Reply:
I changed it here and everywhere I found it. Also, I changed the respective code for `create_rtd`, so that the text is scanned for `$`, not for `\f$` anymore. I hope there are not too many `$` for other reasons than equations.